docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:16.04
```

Comment:
Isn't it better to have the Docker file in the root of the project? I thought it would be easier to register at https://hub.docker.com/ if it was in the root...

Reply:
You mean like this? https://hub.docker.com/r/sklukin/app-mojopaste/

Reply:
I guess you're right. I still want it in the root though, unless you have a good argument for why it should be in `docker/`.